pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "wheel", "cython"]
+requires = ["setuptools", "wheel", "cython", "pkgconfig"]
```

Comment:
Sadly, this is not enough to make all the CI workflows work. You will need to add pkgconfig to the packages installed using apt and yum in `.github/workflows/debian.yml` and `.github/workflows/dist.yml`, respectively.

Reply:
Thanks.